src/fedora/30/amd64/Dockerfile

```diff
@@ -39,3 +39,8 @@ RUN dnf install -y \
         lttng-ust-devel \
         uuid-devel \
     && dnf clean all
+
+# Dependencies of source-build.
+RUN dnf install -y \
+        glibc-langpack-en \
```

Comment:
I suspect this might be better off in the very first layer, along with `clang` and `cmake`. The locale warnings would be printed out at that level too.